addons/event/models/event_event.py

```diff
@@ -177,9 +181,12 @@ def _default_description(self):
     event_registrations_sold_out = fields.Boolean(
         'Sold Out', compute='_compute_event_registrations_sold_out', compute_sudo=True,
         help='The event is sold out if no more seats are available on event. If ticketing is used and all tickets are sold out, the event will be sold out.')
-    start_sale_date = fields.Date(
+    start_sale_datetime = fields.Datetime(
         'Start sale date', compute='_compute_start_sale_date',
         help='If ticketing is used, contains the earliest starting sale date of tickets.')
+    start_sale_date_located = fields.Char(
```

Comment:
Is it still used ? It does not seem to be used anymore in the diff (only defined + compute method).

Reply:
After looking at the code it appears that indeed that field is no longer used, it has now been deleted.